sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -198,6 +198,10 @@ const Lottery = (props: { listing: Listing | undefined }) => {
         (event) => event.type === ListingEventsTypeEnum.publicLottery
       )
       if (listing.lotteryStatus === LotteryStatusEnum.releasedToPartners) {
```

Comment:
Alternative to using .findLast since it was causing build errors. Seems like it was only added recently so our versions may not be quite updated for it https://github.com/microsoft/TypeScript/issues/48829